views/_footer.erb

```diff
@@ -0,0 +1,8 @@
+<p>
```

Comment:
Use Sinatra's "layout" feature for the header and footer:

- http://stackoverflow.com/questions/4190297/how-to-make-a-layout-template-in-sinatra
- http://ruby-log.blogspot.com/2013/02/sinatra-layouts.html